src/Content/Smilies.php

```diff
@@ -294,7 +294,7 @@ private static function heartReplaceCallback(array $matches): string
 	 */
 	public static function isEmojiPost(string $body): bool
 	{
-		$conv = html_entity_decode(preg_replace('#\s#', '', $body));
+		$conv = preg_replace('#\s#', '', html_entity_decode($body));
```

Comment:
Just tested with `&nbsp;` and it needs this modifier to catch the UTF-8 non-breaking space character in the subject:
```suggestion
		$conv = preg_replace('#\s#u', '', html_entity_decode($body));
```